doc/source/release-notes/Percona-XtraDB-Cluster-5.6.27-25.13.rst

```diff
@@ -0,0 +1,78 @@
+
+* Fixed crashing of ``INSERT .... SELECT`` for MyISAM table with ``wsrep_replicate_myisam=ON``. This was caused by TOI being invoked twice when source and destination tables were MyISAM.
+
+* Fixed crash when caching write-set data beyond configured limit. This was caused by TOI flow failing to consider/check error resulting from limit enforcement.
+
+* Fixed error when loading MyISAM table from schema temporary table (with ``wsrep_replicate_myisam=ON``). This was caused by temporary table lookup being done using ``get_table_name()``, which could be misleading as ``table_name`` for temporary tables is set to temporary generated name. Original name of the table is part of ``table_alias``. The fix corrected condition to consider both ``table_name`` and ``alias_name``.
+
+* Fixed error when changing ``wsrep_provider`` in middle of a transaction or as part of procedure/trigger. This is now blocked to avoid inconsistency.
+
+* Removed support for ``innodb_fake_changes`` variable.
+
+* Fixed TOI state inconsistency caused by ``DELAYED_INSERT`` on MyISAM table (``TOI_END`` was not called). Now the ``DELAYED_`` qualifier will be ignored and statement will be interpreted as normal ``INSERT``.
+
+* Corrected locking semantics with FTWRL. It now avoids freeing inheritted lock if follow-up ``FLUSH TABLE`` statement fails. Only frees self-acquired lock.
+
+* ``wsrep_on`` is now a session only variable. That means toggling it will not affect other clients connected to said node. Only the session/client modifying it will be affected. Trying to toggle ``wsrep_on`` in the middle of a transaction will now result in an error. Trx will capture the state of ``wsrep_on`` during start and will continue to use it. Start here means when the first logical changing statement is executed within transaction context.
+
+* Fixed crash caused by ``GET_LOCK`` + ``drupal_workaround``. ``GET_LOCK`` path failed to free all instances of user-level locks after it inherited ``multiple-user-locks`` from PS. The cleanup code now removes all possible references of locks.
+
+* Corrected DDL and DML semantics for MyISAM:
+
+  * DDL (CREATE/DROP/TRUNCATE) on MyISAM will be replicated irrespective of ``wsrep_replicate_miysam`` value
+  * DML (INSERT/UPDATE/DELETE) on MyISAM will be replicated only if ``wsrep_replicate_myisam`` is enabled
+  * SST will get full transfer irrespective of ``wsrep_replicate_myisam`` value (it will get MyISAM tables from donor if any)
+  * Difference in configuration of ``pxc-cluster`` node on ``enforce_storage_engine`` front may result in picking up different engine for same table on different nodes
+  * CTAS statements use non-TOI replication and are replicated only if there is involvement of InnoDB table that needs trx (involvement of MyISAM table will cause CTAS statement to skip replication)
+
+Known Bugs
```

Comment:
Known Issue/Limitation may be good naming for this.
These are not bug as such but software has been designed accordingly.